tokens.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "NOAM",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/noam_logo.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/noam_logo.png",
+        "symbol": "NOAM",
+        "account": "NoamToken",
```

Comment:
@niche500 this is not a valid EOS account name

Please provide a valid token contract name